openapi/components/schemas/ReportJournal.yaml

```
@@ -0,0 +1,27 @@
+type: array
+items:
+  type: object
+  properties:
+    aggregationValue:
+      description: The aggregation field value.
+      type: string
+    bookedMonth:
+      description: Revenue booked month.
```

nitpick: suggestion to aid clarity
```suggestion
      description: Month in which revenue is booked.
```